src/Home/faq.md

```diff
@@ -19,7 +19,7 @@ The only real difference is that Roblox's Vulkan is native, which means there is
 
 ### Why Go?
 
-We chose Golang for its library management, performance, and ease-of-programming. As most of Vinegar's development team contributed to Grapejuice previously, we encountered major difficulties due to its choice of language, Python. For instance, packaging Python libraries was extremely tedious; Grapejuice relied on pip being able to connect to the internet, a privilege not available in Flatpak packaging. Additionally, Grapejuice relied on a highly complex internal build system, which produced several files in various locations, while Golang can output single executables. As a compiled language, Golang also offers superior performance to interpreted languages like Grapejuice, helping Vinegar launch quicker. Golang is also easier to program than Python; we find that semi-static typing and lack of classes helps keep our code linear and easily readable.
+We chose Golang for its library management, performance, and ease-of-programming. As most of Vinegar's development team contributed to Grapejuice previously, we encountered major difficulties due to its choice of language, Python. For instance, packaging Python libraries was extremely tedious; Grapejuice relied on pip being able to connect to the internet, a privilege not available in Flatpak packaging. Additionally, Grapejuice relied on a highly complex internal build system, which produced several files in various locations, while Golang can output single executables. As a compiled language, Golang also offers superior performance to interpreted languages like Python, the language that Grapejuice uses, helping Vinegar launch quicker. Golang is also easier to program than Python; we find that semi-static typing and lack of classes helps keep our code linear and easily readable.
```

Comment:
No reason to point out Python being the language grapejuice uses twice. It's already mentioned in the first paragraph.

```suggestion
We chose Golang for its library management, performance, and ease-of-programming. As most of Vinegar's development team contributed to Grapejuice previously, we encountered major difficulties due to its choice of language, Python. For instance, packaging Python libraries was extremely tedious; Grapejuice relied on pip being able to connect to the internet, a privilege not available in Flatpak packaging. Additionally, Grapejuice relied on a highly complex internal build system, which produced several files in various locations, while Golang can output single executables. As a compiled language, Golang also offers superior performance to interpreted languages, helping Vinegar launch quicker. Golang is also easier to program than Python; we find that semi-static typing and lack of classes helps keep our code linear and easily readable.
```